docs/CHANGELOG.rst

```diff
@@ -16,15 +16,31 @@ Doctest run in CI.
 Changed
 ~~~~~~~
 copy_dir job will now overwrite the target by default
+- Nested support for ``prepros.Serial``
```

Comment:
should we consider these changes as breaking ?

Reply:
indeed!